easybuild/easyconfigs/b/bokeh/bokeh-1.0.4-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'PythonBundle'
+
+name = 'bokeh'
+version = '1.0.4'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://github.com/bokeh/bokeh'
+description = "Statistical and novel interactive HTML plots for Python"
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+dependencies = [
+    ('Python', '3.6.6'),
+    ('PyYAML', '3.13', versionsuffix),
+]
+
+use_pip = True
+
```

Comment:
Please also add `download_dep_fail = True`

Reply:
@vanzod `download_dep_fail = True` is already the default in `PythonBundle` easyblock.